project/src/test/java/com/google/sps/SearchDataServletTest.java

```diff
@@ -25,27 +42,83 @@
   private HttpServletResponse secondPostResponse;
   private HttpServletRequest getRequest;
   private HttpServletResponse getResponse;
-  private StringWriter stringWriter;
-  private PrintWriter printWriter;
+  private StringWriter getStringWriter;
+  private StringWriter postStringWriter;
+  private StringWriter secondPostStringWriter;
+  private PrintWriter getPrintWriter;
+  private PrintWriter postPrintWriter;
+  private PrintWriter secondPostPrintWriter;
   private SearchDataServlet searchDataServlet;
+  private EventCreationServlet eventCreationServlet;
+  private KeywordHelper mockKeywordHelper;
   private static final String EVENT_ID_1 = TestUtils.newRandomId();
   private static final String EVENT_ID_2 = TestUtils.newRandomId();
   private static final String PARAMETER_KEYWORD = "keyword";
   private static final String PARAMETER_EVENT_ID = "event-id";
+  private static final String PARAMETER_NAME = "name";
   private static final String PARAMETER_DESCRIPTION = "description";
-  private static final String WALKING = "walking";
   private static final String CAKE = "cake";
-  private static final String MUSIC = "music";
-  private static final String DESCRIPTION_WITH_MUSIC_KEYWORD =
-      "This is an annual event held at the State Capitol to celebrate agricultures."
-          + "The event will include food, live music and educational booths with public speakers."
-          + "Event hours is 9 am- 1 pm. 2550 attendees anticipated.";
-  private static final String DESCRIPTION_WITH_WALKING_KEYWORD =
-    "Sutter Middle School will be walking to McKinley Park. 7th grade class and teachers"
-    + " will have a picnic, play games and eat lunch at the park and Clunie Pool.";
-  private static final String DESCRIPTION_WITHOUT_CAKE_KEYWORD =
-    "Sutter Middle School will be walking to McKinley Park. 7th grade class and teachers"
-    + " will have a picnic, play games and eat lunch at the park and Clunie Pool.";
+  private static final String GAMES = "games";
+  private static final String NAME_WITHOUT_GAMES = "End of the Year Picnic";
+  private static final String DESCRIPTION_WITHOUT_GAMES =
+      "Sutter Middle School will be walking to McKinley Park. 7th grade class and teachers will"
+          + " have a picnic and eat lunch at the park and Clunie Pool.";
+  private static final String NAME_WITH_GAMES = "End of the Year Picnic and Games";
+  private static final ArrayList<Keyword> KEYWORDS_NAME_WITHOUT_GAMES =
```

Comment:
HIGH_RELEVANCE ? 
I guess my question with these tests is like KEYWORDS_NAME_WITHOUT_GAMES (with no relevance labeled) 'medium relevance' or does the relevance not matter in these?